docs/install.rst

```diff
@@ -188,15 +173,28 @@ Fedora                               Mohamed El Morabity <melmorabity at fedorap
 Gentoo                               soredake <fdsfgs at krutt.org>
 NetBSD                               Maya Rashish <maya at netbsd.org>
 NixOS                                Tuomas Tynkkynen <tuomas.tynkkynen at iki.fi>
-OpenBSD                              Brian Callahan <bcallah at openbsd.org>
 openSUSE                             Simon Puchert <simonpuchert at alice.de>
 Solus                                Joey Riches <josephriches at gmail.com>
-Ubuntu                               Alin Andrei <andrew at webupd8.org>
 Void                                 Michal Vasilek <michal at vasilek.cz>
 Windows binaries                     Sebastian Meyer <mail at bastimeyer.de>
+Linux AppImages                      Sebastian Meyer <mail at bastimeyer.de>
 ==================================== ===========================================
 
 
+Package availability
+--------------------
+
+Packaging is not done by the Streamlink maintainers themselves except for
+the `PyPI package <PyPI package and source code_>`_,
+the `Windows installers + portable builds <Windows binaries_>`_,
+and the `Linux AppImages <AppImages_>`_.
+
+If a package is not available for your operating system / distro or your system's architecture, or if it's out of date
+or broken, then please contact the respective package maintainers or package-repository maintainers.
+
+Please open an issue or pull request on GitHub if an available and maintained package is missing from the install docs.
```

Comment:
Do we even want to say this? I feel like people will interpret this incorrectly and open issues because packages are missing any way. Can we make the assumption that package maintainers know what to do?

Reply:
Package maintainers usually don't know or don't care about external documentation. What they care about is the availability of software in their package repos.

I've reworded the section a bit and put more emphasis on when to open issues / PRs for unlisted packages.